esp/esp/program/modules/handlers/programprintables.py

```diff
@@ -2038,3 +2073,26 @@ def __init__(self, program, *args, **kwargs):
 
         self.converter = AllClassesFieldConverter(program)
         self.fields['subject_fields'].choices = self.converter.field_choices
+
+class StudentScheduleFormatForm(forms.Form):
+    schedule_fields = forms.MultipleChoiceField(choices = [("username", "Student's Username"),
```

Comment:
It's forcing me to select at least one field, which I feel like shouldn't be necessary